ocaml-lsp-server/src/ocamlformat_rpc_lib.ml

```diff
@@ -0,0 +1,220 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                              OCamlFormat                               *)
+(*                                                                        *)
+(*            Copyright (c) Facebook, Inc. and its affiliates.            *)
+(*                                                                        *)
+(*      This source code is licensed under the MIT license found in       *)
+(*      the LICENSE file in the root directory of this source tree.       *)
+(*                                                                        *)
+(**************************************************************************)
```

Comment:
If this is vendored code, it should live in the vendored directory.

Reply:
It should disappear before we merge the PR. But I will move it to the vendor dir if it doesn't at the end.

Reply:
Sounds good.